content/applications/inventory_and_mrp/inventory/management/reporting/inventory_valuation_config.rst

```diff
@@ -82,27 +82,184 @@ Costing method
+             So, `$120 / 12 = $12`.
+
+         - When delivering ten products, the average **Unit Cost** is used to calculate the
+           **Inventory Value**, regardless of the purchase price of the product. Therefore,
+           Inventory Value is `$144 + (-10 * $12)` = `$144 - $120` = `$24`
+
+         - Receive two products for $6 each:
+
+           - **Inventory Value**: `$24 + (2 * $6)` = `$24 + $12` = `$36`
+           - **Unit Cost**: `$36 / 4 = $9`
+
+      .. note::
+         When choosing :guilabel:`Average Cost (AVCO)` as the :guilabel:`Costing Method`, changing
+         the numerical value in the :guilabel:`Cost` field for products in the respective product
+         category creates a new record in the *Inventory Valuation* report to adjust the value of
+         the product. The :guilabel:`Cost` amount will then automatically update based on the
+         average purchase price both of inventory on hand and the costs accumulated from validated
+         purchase orders.
+
+   .. tab:: First In First Out (FIFO)
+
+      Tracks the costs of incoming and outgoing items in real-time and uses the real price of the
+      products to change the valuation. The oldest purchase price is used as the cost for the next
+      good sold until an entire lot of that product is sold. When the next inventory lot moves up in
+      the queue, an updated product cost is used based on the valuation of that specific lot.
+
+      This method is arguably the most accurate inventory valuation method for a variety of reasons,
+      but, it is highly sensitive to input data and human error.
+
+      .. list-table::
+         :header-rows: 1
+         :stub-columns: 1
+
+         * - Operation
+           - Unit Cost
+           - Qty On Hand
+           - Incoming Value
+           - Inventory Value
+         * -
+           - $0
+           - 0
+           -
+           - $0
+         * - Receive 8 products for $10/unit
+           - $10
+           - 8
+           - 8 * $10
+           - $80
+         * - Receive 4 products for $16/unit
+           - $12
+           - 12
+           - 4 * $16
+           - $144
+         * - Deliver 10 products
+           - $16
+           - 2
+           - | -10 * $12
+             | -2 * $16
+           - $32
+         * - Receive 2 products for $6/unit
+           - $11
+           - 4
+           - 2 * $6
+           - $44
+
+      .. spoiler:: How are Unit Cost and Inventory Value at each step calculated?
+
+         - When receiving four products for $16 each (process is similar to the average costing
+           method in this step):
+
+           - **Inventory Value** is calculated by adding the previous inventory value with the
+             incoming value. So, with `$80 + (4 * $16)` = `$80 + $40` = `$120`.
+           - **Unit Cost** is calculated by dividing the inventory value by the quantity on hand.
+             So, `$120 / 12 = $12`.
```

Comment:
I think, with fixed math (see my previous comment), this equation should now read "$140/12 = $12"